src/config/rounds/round3.tsx

```diff
@@ -16,6 +17,7 @@ export const round3: RoundDescription<Round3ActionId> = {
       in the next Sprint.
     </p>
   ),
+  effect: () => ({ title: EFFECT_HIDDEN, gremlinChance: 50 }),
```

Comment:
👆 IMPORTANT

effects can now change the chance that gremlins might occur. This is currently the only one creating a 50/50 chance from round 3 on.